src/checkTargetMatchToPR.js

```diff
@@ -11,11 +12,23 @@ const checkTargetMatchToPR = (prTitle, target) => {
   if (!match) {
     return true
   }
-  const diff = semverDiff(match[1], match[2])
+
+  const [, from, to] = match
+
+  if(hasBadChars(from) || hasBadChars(to)) {
+    return false
```

Comment:
let's log something when this happens. also not clear to me what this means to be honest